mne/forward/tests/test_field_interpolation.py

```diff
@@ -222,3 +224,29 @@ def test_as_meg_type_evoked():
     assert_true(np.corrcoef(data1, data2)[0, 1] > 0.95)
 
 run_tests_if_main()
+
+
+@testing.requires_testing_data
+def test_transform_instances():
+    evoked = read_evokeds(evoked_fname, condition='Left Auditory')
+    info = evoked.info
+    evoked_rot = evoked.copy()
+    info_rot = evoked_rot.info
+    picks = pick_types(info_rot, meg=True)
+    names = np.array(info_rot['ch_names'])[picks]
+
+    rot = rotation(x=0, y=np.pi/12, z=0)
```

Comment:
I'd really check if the coils are getting rotated by the desired angle and in the desired plane. Feel free to share the plots